mypy/parse.py

```diff
@@ -1845,7 +1856,8 @@ def parse_error_at(self, tok: Token, skip: bool = True) -> None:
         elif isinstance(tok, Indent) or isinstance(tok, Dedent):
             msg = 'Inconsistent indentation'
         else:
-            msg = 'Parse error before {}'.format(token_repr(tok))
+            reason = " due to: {}".format(reason) if reason else ""
```

Comment:
I'd remove "due to" here.  I think the ":" alone corresponds a bit better to standard error messaging.